Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1506,6 +1510,31 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
     return false;
 }
 
+bool UnifiedPDFPlugin::performCopyEditingOperation() const
+{
+#if PLATFORM(MAC)
+    if (!m_currentSelection)
+        return false;
+
+    if (![m_pdfDocument allowsCopying]) {
+        [[NSNotificationCenter defaultCenter] postNotificationName:get_PDFKit_PDFViewCopyPermissionNotification() object:nil];
```

Comment:
I created a `WKPDFFormMutationObserver` earlier to use with NSNotificationCenter messages. Maybe we could think about reusing it with a more general name here or a future addition if we don't think it's super important right now

Reply:
It looks like `WKPDFFormMutationObserver` is used to listen to messages. Is there value in reusing it when we want to send a notification? How would we reuse it?